mne/beamformer/_dics.py

```diff
@@ -117,37 +444,44 @@ def dics(evoked, forward, noise_csd, data_csd, reg=0.05, label=None,
               will result in complex time courses, in which case absolute
               values will be returned.
 
-    .. note:: This implementation has not been heavily tested so please
-              report any issues or suggestions.
+    .. warning:: This implementation has not been heavily tested so please
+                 report any issues or suggestions.
 
     Parameters
     ----------
     evoked : Evoked
         Evoked data.
     forward : dict
         Forward operator.
-    noise_csd : instance of CrossSpectralDensity
-        The noise cross-spectral density.
-    data_csd : instance of CrossSpectralDensity
+    csd : instance of CrossSpectralDensity
```

Comment:
This is a backward-incompatible change. Not sure what we should do about it. Maybe have `noise_csd=None`? Or maybe make an exception to the deprecation rule here since results will probably change quite a bit anyway?

Reply:
... actually maybe we should just deprecate `dics` and tell people to use the two-step `make_dics` and `apply_dics`. This mirrors what we do with minimum norm anyway.

Reply:
The results will be completely different. Since the DICS implementation has been re-build from the group up, I don't think we can keep backwards compatibility in a meaningful way. Also, @dengemann gave me permission to break all the things :)

Reply:
@larsoner just for completeness: for the LCMV beamformer, we also have ``lcmv``, ``make_lcmv``, and ``apply_lcmv`` with ``lcmv`` being a wrapper for the other two. 
(Originally, it was only implemented as ``lcmv``, the separate functions for fitting and applying the beamformer were only added this summer.)